src/applications/appeals/shared/components/IssuesSubmitted.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const IssuesSubmitted = props => (
```

Comment:
similar to above, not blocking but you could destructure here and then do `issues` in the `<ul>` below

```suggestion
export const IssuesSubmitted = ({ issues }) => (
```